### PR TITLE
Fix docstrings

### PR DESCRIPTION
**Pull request by aadcg**

### CODE REVIEW

File: source/buffer.lisp [hunk 1]
```diff
@@ -1649,8 +1649,8 @@ When there is no next buffer, go to the first one so as to cycle."
                    (set-current-buffer next))))
 
 (define-command switch-buffer-last ()
-  "Switch to the last showing buffer in the list of buffers.
-That is to say, the one with the most recent access time after the current buffer."
+  "Switch to the last visited buffer.
+I.e., the one with the most recent access time."
```
Comment (Ambrevar) [inline source/buffer.lisp:1653]: I recommend we refrain from this academic initials: they are not necessarily well understood by everyone. (Plus somewhat elitist.)

`I.e.` translate into "That is,".

File: source/mode/os-package-manager.lisp [hunk 2]
```diff
@@ -381,7 +381,7 @@ OBJECTS can be a list of packages, a generation, etc."
     buffer))
 
 (define-command-global switch-os-generation ()
-  "Switch generation of selected profile."
+  "Switch generation of selected profiles."
```
Comment (Ambrevar) [inline source/mode/os-package-manager.lisp:384]: I believe you can select only 1 profile here.

File: source/mode/os-package-manager.lisp [hunk 3]
```diff
@@ -397,7 +397,7 @@ OBJECTS can be a list of packages, a generation, etc."
                         profile generation)))
 
 (define-command-global delete-os-generations ()
-  "Delete generations of selected profile."
+  "Delete generations of selected profiles."
```
Comment (Ambrevar) [inline source/mode/os-package-manager.lisp:400]: Same.

**Comment by Ambrevar** [comment]
Excellent, thanks for taking the pain to go through all this!

**Comment by jmercouris** [comment]
I can fix the discrepancies pointed out since Andre will be AFK.

**Comment by aadcg** [comment]
@jmercouris that would be appreciated :)

**Comment by aadcg** [comment]
> Excellent, thanks for taking the pain to go through all this!

No pain at all!

**Pull request merged**